docs/webrtc/nanostream_webrtc_quality.md

```diff
@@ -0,0 +1,96 @@
+---
+id: nanostream_webrtc_quality
+title: Quality Settings
+sidebar_label: Quality Settings
+---
+
+# Configuring stream quality
+
+Webcasting quality can be configured by setting different stream encoding and transcoding properties.
+In the nanostream Webcaster this is basically performed in three steps:
+
+1) Previewing a stream. Certain stream properties can be set when starting the preview: <br>
+[Video and audio preview](./nanostream_webrtc_quality#video-and-audio-preview) 
+2) Encoding the stream in the browser. Upstream bitrates can be configured: <br>
+[Encoding bitrates](./nanostream_webrtc_quality#encoding-bitrates)
+3) Sending the stream to the nanostream cloud. Audio transcoding is done on server side: <br>
+[Transcoding on the server](./nanostream_webrtc_quality#transcoding-on-the-server)
+
+ 
+ ## Video and audio preview 
+
+Streams have different properties that can be set when starting a preview.<br>
+Those are:
+
+- video source (camera or screen share)
+- video resolution
+- video framerate
+- audio source
+
+The regarding API call is [startPreview()](./nanostream_webrtc_api#startpreviewconfig)
```

Comment:
Could it be better to use the word `corresponding` instead of `regarding`..?

Reply:
sure, updated